Test1/src/leetcode/LeetCode17.java

```diff
@@ -0,0 +1,70 @@
+package leetcode;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * https://leetcode.com/problems/letter-combinations-of-a-phone-number/
+ * @author minibig
+ */
+public class LeetCode17 {
+
+	public List<String> letterCombinations(String digits) {
+		
+		List<String> list = new ArrayList<String>();
+		
+		if(digits.isEmpty()) return list;
+		
+		Map<Integer, String> digitMap = new HashMap<Integer, String>();
+		
+		digitMap.put(2, "abc");
+		digitMap.put(3, "def");
+		digitMap.put(4, "ghi");
+		digitMap.put(5, "jkl");
+		digitMap.put(6, "mno");
+		digitMap.put(7, "pqrs");
+		digitMap.put(8, "tuv");
+		digitMap.put(9, "wxyz");
```

Comment:
Usually for this kind of objects we'll make it a static object and construct it within a static class.

The reason is that you'll probably be able to reuse the same object at many places but apparently you don't want to construct it again and again.